### ISSUE TITLE
Curious, why the limit on the number of icons per project?

### ISSUE THREAD

**Issue by jlarmstrongiv** [iconhub-io/web#3]
The license states:
```
- Use in unlimited number of commercial project as long as you use less than 20 unique icons
- Use in unlimited number of personal/non profit project, as long as you use less than 50 unique icons
```
I’m curious why those lines are there when I also can’t:
```
- Sell or redistribute the icons in any manner
```
Also, there’s no way to purchase a better license to let me use more icons.  When I build a website, there’s icons in the navigation sidebar, the media players, the featured section, etc.  More than 20.  Maybe 50.  Having to worry about an icon quota means I probably won’t use the project and it’s not really “free”.

**tl;dr** I like the really project and your icons, especially the styles.  The limit means I can’t use them (and I have no way to increase the limit through purchasing or donating).

**Comment by uyab** [comment]
cc @owwstudio

**Comment by jlarmstrongiv** [comment]
Any updates @uyab and @owwstudio? 😄
